### PR TITLE
Add new icons; fix the contributing procedure

### PR DESCRIPTION
**Pull request by ethanph5**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -15,17 +15,17 @@ module.exports = function(grunt) {
...
-  })
+  });
 
   grunt.loadNpmTasks('grunt-webfont');
-  grunt.loadNpmTasks('grunt-git');
+  grunt.loadNpmTasks('grunt-contrib-clean');
 
-  grunt.registerTask('default', ['gitrm', 'webfont'])
-}
+  grunt.registerTask('default', ['clean', 'webfont']);
```
Comment (Shwetank87) [inline Gruntfile.js]: so we are just deleting files now? and onus of adding them to the commit lies with the coder?

File: README.md [hunk 2]
```diff
@@ -11,9 +11,8 @@ This plugin requires Grunt 0.4.
 ###OS X
 
 ```
-brew install ttfautohint --with-python
+brew install fontforge
```
Comment (Shwetank87) [inline README.md]: :cookie:

File: dist/_kx-icon-font.scss [hunk 3]
```diff
@@ -19,25 +19,31 @@
...
-.kx-icon-notifications:before { content: "\f110"; }
-.kx-icon-pin:before { content: "\f111"; }
-.kx-icon-population:before { content: "\f112"; }
-.kx-icon-report-h:before { content: "\f113"; }
-.kx-icon-report:before { content: "\f114"; }
-.kx-icon-standard-segment-h:before { content: "\f115"; }
-.kx-icon-standard-segment:before { content: "\f116"; }
-.kx-icon-supertag:before { content: "\f117"; }
-.kx-mail:before { content: "\f118"; }
+.kx-ico-st-composite:before { content: "\f103"; }
```
Comment (Shwetank87) [inline dist/_kx-icon-font.scss]: can we be consistent with the prefix? kx-ico and kx-icon are kind of aggravating.

**Comment by Shwetank87** [comment]
LGTM.. however please fix the naming prefix. rest looks good. Thanks for fixing the read me :)

File: Gruntfile.js (hunk 1 above)
Comment (ethanph5) [inline Gruntfile.js]: I really don't see the benefit of using `gitrm`. It generate warning and stop working if you run the command again.

Comment (Shwetank87) [inline Gruntfile.js]: I agree. was just confirming the behaviour

**Pull request merged**